### PR TITLE
BaSP-TG-41: Added "Populate" to POST and PUT method.

### PR DESCRIPTION
**Pull request by Eju97**
Added populate to POST and PUT method by QA request.

### CODE REVIEW

File: src/controllers/time-sheets.js [hunk 1]
```diff
@@ -57,7 +57,11 @@ export const createTimeSheet = async (req, res) => {
       employee: req.body.employee,
       project: req.body.project,
     });
-    const result = await newTimeSheet.save();
+    await newTimeSheet.save();
+    const result = await TimeSheets.find(newTimeSheet)
```
Comment (lmalvasia) [inline src/controllers/time-sheets.js:61]: Not sure if this is the best way to populate an entity after creating it. Could we do something like?
```
    const result = await newTimeSheet.save();
    const populatedResult = await result.populate('task')
      .populate('employee')
      .populate('project');
    return res.status(201).json({
      message: 'Time sheet created successfully',
      data: populatedResult,
      error: false,
    });
```

Comment (Eju97) [reply src/controllers/time-sheets.js:61]: image
I have tried the solution you suggested but it returns an error.

Comment (Eju97) [reply src/controllers/time-sheets.js:61]: Even tho it creates the timesheet

Comment (Eju97) [reply src/controllers/time-sheets.js]: Solved!

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**